qir/qat/Apps/Qat/Qat.cpp

```diff
@@ -59,6 +59,7 @@
 
 #include <dlfcn.h>
 
+#include <climits>
```

Comment:
Now that `cfg.inlineParameter()` is used below, this include is no longer needed.

Reply:
Removed.